site/docs/components/AppDemo.tsx

```diff
@@ -0,0 +1,118 @@
+'use client';
+import { OnchainKitProvider } from '@coinbase/onchainkit';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import type { ReactNode } from 'react';
+import { http, WagmiProvider, createConfig } from 'wagmi';
+import { base, baseSepolia } from 'wagmi/chains';
+import { coinbaseWallet } from 'wagmi/connectors';
+import '@coinbase/onchainkit/styles.css';
+import { createWalletClient } from 'viem';
+import { privateKeyToAccount } from 'viem/accounts';
+import { useTheme } from '../contexts/Theme.tsx';
+
+const queryClient = new QueryClient();
+
+// WARNING: This is a publicly visible demo/test account with absolutely no real assets or value.
+// It is used ONLY for documentation examples and should NEVER be used in production or to store any assets.
+const DUMMY_PK =
+  '0xa67bdd8a49324aa36cb3f7f7064b9b560e3aa653b774be9793415c0a6fc62cf8';
+
+const demoWalletConnector = () => () => {
```

Comment:
nit: i find chained arrow functions pretty hard to reason about. consider making the outer function a named function, and return the inner arrow function